utils.py

```diff
@@ -106,3 +117,28 @@ def check_disabled_component(uni_a: UNI, uni_z: UNI):
     if uni_z.interface.status == EntityStatus.DISABLED:
         id_ = uni_z.interface.id
         raise DisabledSwitch(f"Interface {id_} is disabled")
+
+
+def make_uni_list(list_circuits: list) -> list:
+    """Make uni list to be sent to sdntrace"""
+    uni_list = []
+    for circuit in list_circuits:
+        if isinstance(circuit.uni_a.user_tag, TAGRange):
+            # TAGRange value from uni_a and uni_z are currently mirrored
+            for mask in circuit.uni_a.user_tag.mask_list:
```

Comment:
`mask_list` currently can be initialized as `None` so this iteration here or in other places, in some cases it might crash. So, on Kytos core we might want to initialize as an empty list if the initial `mask_list` was `None`.

Reply:
`mask_list` initialize value change in last kytos [commit](https://github.com/kytos-ng/kytos/pull/428/commits/f80d202828284e2a229937ad1eba7356edf4a049#).